tests/unit/s2n_aead_aes_test.c

```diff
@@ -187,7 +188,7 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->header_in));
             EXPECT_SUCCESS(s2n_stuffer_copy(&conn->out, &conn->header_in, 5));
             EXPECT_SUCCESS(s2n_stuffer_copy(&conn->out, &conn->in, s2n_stuffer_data_available(&conn->out)));
-            conn->in.blob.data[5 + j] ++;
```

Comment:
How were these tests passing previously with the wrong indexing? Does the indexing not matter, or do we need to be checking for a more specific error from s2n_record_parse?

Reply:
These set of tests were iteratively flipping each bit of the packet (but not the MAC tag itself) to verify that MAC verification would fail. The wrong indexing either meant that some bits were flipped more than once or that some bits of the MAC tag would be flipped. Either way, MAC verification would still fail. The indexing was not off by an amount that would cause reading past the buffer.